app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderAdapter.kt

```diff
@@ -41,9 +42,9 @@ class PreviewSliderAdapter(
         val args = PreviewFragmentArgs(fileId = file.id, userDrive = userDrive).toBundle()
 
         return when (file.getFileType()) {
-            ExtensionType.IMAGE -> PreviewPictureFragment()
-            ExtensionType.VIDEO -> PreviewVideoFragment()
-            ExtensionType.AUDIO -> PreviewMusicFragment()
+            ExtensionType.IMAGE -> PreviewPictureFragment(isPublicShared)
+            ExtensionType.VIDEO -> PreviewVideoFragment(isPublicShared)
+            ExtensionType.AUDIO -> PreviewMusicFragment(isPublicShared)
```

Comment:
For fragments, don't pass parameters like that because it's dangerous and creates bugs after recreate. Instead, use `Fragment.arguments`.